Automattic/wp-calypso
# Remove code added for the premium bump experiment

#### Changes proposed in this Pull Request

* This PR removes the code added for the Personal to Premium plan upgrade offer experiment, introduced in https://github.com/Automattic/wp-calypso/pull/47948.

#### Testing instructions



* Signup for a new account and verify that:
   - Purchasing a Personal, Business, or eCommerce plan will show the Quick Start upsell(if you are assigned to the treatment of the `conciergeUpsellDial` test)
   -  Purchasing a Premium plan shows a Business bump offer
